### PR TITLE
Add summer school 2021

### PR DESCRIPTION
**Pull request by frankharkins**
This is now ready as MVP.

[Link to deploy](https://platypus-review-pr-138.herokuapp.com/course/qgss2021)

### CODE REVIEW

File: frontend/scss/custom.scss [hunk 1]
```diff
@@ -56,3 +56,17 @@ button span.rangle {
...
   }
 }
+
+/* Make youtube full width */
+.youtube-wrapper {
+  position: relative;
+  width: 100%;
+  height: 0;
+  padding-bottom: 56%;
+}
```
Comment (vabarbosa) [inline frontend/scss/custom.scss:66]: @frankharkins you may want to add `margin-bottom` to give the a bit more room between the video and the next paragraph:

image

**Review by vabarbosa — CHANGES_REQUESTED** [review]
@frankharkins thank you! i have one comment about the spacing between the video and surrounding paragraphs

**Review by techtolentino — APPROVED** [review]
This looks great.

Thanks @frankharkins

File: frontend/scss/custom.scss (hunk 1 above)
Comment (frankharkins) [reply frontend/scss/custom.scss]: Thank you!

**Pull request merged**